src/CONST.js

```diff
@@ -463,6 +463,7 @@ const CONST = {
                 CLOSED: 'CLOSED',
                 CREATED: 'CREATED',
                 TASKEDITED: 'TASKEDITED',
+                TASKCANCELED: 'TASKCANCELED',
```

Comment:
beep boop 🤖 🐧 

This PR caused https://github.com/Expensify/App/issues/21314#issuecomment-1603797430

Because we have a typo-ish here. 
The backend uses `TASKCANCELLED` (notice the double LL)